Client/mods/deathmatch/logic/CClientPed.cpp

```diff
@@ -2737,7 +2737,7 @@ void CClientPed::StreamedInPulse ( bool bDoStandardPulses )
                 // Copy our name incase it gets deleted
                 SString strAnimName = m_strAnimationName;
                 // Run our animation
-                RunNamedAnimation ( m_pAnimationBlock, strAnimName, m_iTimeAnimation, m_bLoopAnimation, m_bUpdatePositionAnimation, m_bInterruptableAnimation, m_bFreezeLastFrameAnimation );
```

Comment:
Hmm, you've got a good point. This'll need some extra testing to make sure this is what we actually want.